### DISCUSSION TITLE
Nuxt DevTools

### DISCUSSION THREAD

**Discussion by danvega** [nuxt/nuxt#17485] [General]
I am trying to find some more information on the upcoming Nuxt DevTools. Is this still in development or is there somewhere I can preview this? I just want to give some highlights and or screenshots for an upcoming presentation but I can't find much on it. Is this going to be a plugin for the new Vue DevTools?

**Comment by pi0** [comment]
Hi dear @danvega. We started with [nuxt/ui](https://github.com/nuxt/ui) project but not implemented anything yet to be able to beta test. If curious, this is planned roadmap for the devtools module:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/5158436/171294091-4ec6a3fa-3c07-4999-a54d-402aa470f252.png">

**Reply by danvega** [reply]
Thank you! I was going to mention it in my talk next week but ill probably leave it out for now. Looking forward to seeing what you guys do with that 👏
